techniques/system/distributePolicy/1.0/rudder-rsyslog-root.st

```diff
@@ -70,3 +70,17 @@ $template RudderReportsFormat,"insert into RudderSysEvents (executionDate, nodeI
 
 ## Syslog messages from "rudder" forwarded from AIX
 :msg, ereregex, "from .*: rudder:" ~
+
+&if(SKIPIDENTIFY)&
+
+$AllowedSender UDP, &ALLOWEDNETWORK&
+$AllowedSender TCP, &ALLOWEDNETWORK&
+
+&else&
+
+&MANAGED_NODES_IP : {ip |
+$AllowedSender UDP, &ip&
```

Comment:
the line can get pretty big there (8 Mo if I understand correctly), or is it one line per ip ?
In the first case, how does the agent handles it (line > 4Ko are usually truncated). In the second case, it is perfect